.eslintrc

```diff
@@ -23,7 +23,6 @@
       "error",
       "array",
     ],
-    "angular/di-order": "error",
```

Comment:
Just to clarify, will these rules will be permanently relaxed?

Reply:
Yes, since we have our own custom linting rule that takes care of it and the style it follows is not according to this rule and we can only follow one of them.

Reply:
okay, sounds good!